src/schema/objects/metadata.yaml

```diff
@@ -3366,6 +3366,20 @@ SubjectArtefactDescription:
     If this field is set to `"n/a"`, it will be interpreted as absence of major
     source of artifacts except cardiac and blinks.
   type: string
+TablePosition:
+  name: TablePosition
+  display_name: Table Position
+  description: |
+    The x, y and z coordinates of the table relative to an implementation
+    specific reference point. The array MUST contain three numeric values
+    corresponding to x, y, and z axis of the coordinate system in that exact
+    order.
```

Comment:
What coordinate system is this position in? Is it DICOM LPS, or is it the responsibility of the converter software to use RAS for consistency with NIfTI?

I don't fully understand "an implementation specific reference point". I suppose it means that the value cannot be meaningfully compared across datasets unless `ManufacturersModelName` and `SoftwareVersions` are the same. Is it correct or incorrect to say that this is the offset from the origin of the image affine to the scanner isocenter?

@neurolabusc Is this enough information for `dcm2niix` to have an unambiguous directive on what to do?

Reply:
> What coordinate system is this position in? Is it DICOM LPS, or is it the responsibility of the converter software to use RAS for consistency with NIfTI?

Since NIfTI is using RAS+ coordinates, I was thinking that to be consistent, it should be using RAS+ as well. I can make it more explicit.

> Is it correct or incorrect to say that this is the offset from the origin of the image affine to the scanner isocenter?

This depends where the origin of the table position is since only the offset is reported. From my point of view, I would make it so that it is the case to be able to know where isocenter is. This implies that we know the origin of the table position (it seems to be at isocenter, at least from my experience on Siemens and Philips), and that the origin of the affine is isocenter if the table position is at its origin. This seems to be my experience.

If we can say with certainty that this is the offset from isocenter, I would go with that. We could even change the name tag to be more explicit (e.g.: affineIsoOffset, ...). Maybe @neurolabusc knows a bit more.

Reply:
> Since NIfTI is using RAS+ coordinates, I was thinking that to be consistent, it should be using RAS+ as well. I can make it more explicit.

Agreed. Given that DICOM is LPS, making it explicit that converters will need to adjust would be good.

> This depends where the origin of the table position is since only the offset is reported. From my point of view, I would make it so that it is the case to be able to know where isocenter is. This implies that we know the origin of the table position (it seems to be at isocenter, at least from my experience on Siemens and Philips), and that the origin of the affine is isocenter if the table position is at its origin. This seems to be my experience.

Got it. What about the following text:

> The table position, relative to an implementation-specific reference point, often the isocenter.
> Values must be an array of three distances in millimeters, respectively
> right, anterior and superior (RAS+) of the image affine.

There's some additional text in [(0018,9327)](https://dicom.innolitics.com/ciods/enhanced-ct-image/enhanced-ct-image-multi-frame-functional-groups/52009229/00189326/00189327):

> Shall be relative to the same reference point for all frames in this SOP Instance, but may be different from the reference point in other SOP Instances. Positions as the table moves into the gantry viewed from the front are more negative.

I'm not sure if/how best to include this concept. Possibly "Shall be relative to the same reference point for all images with the same acquisition parameters."? Or maybe best to leave out.

Reply:
Great ideas, I will update the PR to reflect these comments. 

> I'm not sure if/how best to include this concept. Possibly "Shall be relative to the same reference point for all images with the same acquisition parameters."? Or maybe best to leave out.

I originally opted to leave it out since the goal is to make it relative to the isocenter.

Reply:
I updated the code to reflect the changes. @neurolabusc, do you have any thoughts on this?

Reply:
The definition looks clear to me.